example/windows/AsyncStorageExample/AsyncStorageExample.vcxproj

```diff
@@ -163,6 +163,7 @@
       npx --no-install react-native bundle --platform windows --entry-file index.js --bundle-output $(MSBuildThisFileDirectory)/Bundle/index.windows.bundle --assets-dest $(MSBuildThisFileDirectory)/Bundle
     </BundleCommand>
   </PropertyGroup>
+  <Target Name="Deploy"/>
```

Comment:
👍 I see this is the same issue [resolved here](https://github.com/react-native-async-storage/async-storage/pull/463)